src/System.Device.Gpio/System/Device/Spi/Devices/UnixSpiDevice.cs

```diff
@@ -181,8 +210,13 @@ public override unsafe void WriteByte(byte value)
         public override unsafe void Write(ReadOnlySpan<byte> buffer)
         {
             Initialize();
+            byte[] toSend = buffer.ToArray();
```

Comment:
I'm ok with this code when we're inverting but in case when we're not we used to not allocate at all here and now we copy the buffer

Reply:
same comment for other similar methods in UnixSpiDevice, on Windows we always allocated anyway